article.tex

```diff
@@ -75,34 +75,34 @@
 
 
 \section[Introduction]{Introduction}
-Before each statistical analysis, data cleaning is often required to ensure a good data quality.
-Data cleaning is the process of detecting errors and inconsistencies in data sets \citep{Chapman_2005}.
+Before each statistical analysis, data cleaning is often required to ensure good data quality.
+Data cleaning is the process of detecting errors and inconsistencies in datasets \citep{Chapman_2005}.
```

Comment:
Interesting, didn't know that. Thanks for highlighting the difference. My intent was just to make it uniform over the whole document.